lib/Transport.js

```diff
@@ -48,6 +48,7 @@ Transport.prototype._action = function(method, url, data, callback) {
       try {
         response = JSON.parse(err.response.text);
       } catch(e) {
+        console.error("Transport Error: " + JSON.stringify(err));
```

Comment:
We should never get here, when we do it means there's either connection issues or the remote API isn't responding with valid JSON. It usually indicates a CORS issue.